localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,344 @@
+ jarldom_vrykul:0 "Jarldom"
+ high_king_male_vrykul:0 "High King"
+ high_queen_female_vrykul:0 "High Queen"
+ high_kingdom_vrykul:0 "High Kingdom"
+
+ ### Frost Vrykul ###*
+ # Feudal
+ thane_male_frost_vrykul:0 "Thane"
+ thane_female_frost_vrykul:0 "Thane"
+ chiefdom_frost_vrykul:0 "Chiefdom"
+ overthane_male_frost_vrykul:0 "Overthane"
+ overthane_female_frost_vrykul:0 "Overthane"
+ jarldom_frost_vrykul:0 "Jarldom"
+ high_king_male_frost_vrykul:0 "High King"
+ high_queen_female_frost_vrykul:0 "High Queen"
+ high_kingdom_frost_vrykul:0 "High Kingdom"
+
+ ### Staroveric ###
+ # Feudal
+ grand_prince_male_solstice:0 "Grand Prince"
+ grand_princess_female_solstice:0 "Grand Princess"
+ grand_principality_solstice:0 "Grand Principality"
+ tsar_male_solstice:0 "Tsar"
+ tsaritsa_female_solstice:0 "Tsaritsa"
+ tsardom_solstice:0 "Tsardom"
+
+ ### Dryad group ###
+ # Feudal / Republic / Clan / Theocratic / Tribal
+ tender_male_dryad_group:0 "Tender"
+ tender_female_dryad_group:0 "Tender"
+ refuge_dryad_group:0 "Refuge"
+ guardian_male_dryad_group:0 "Guardian"
+ guardian_female_dryad_group:0 "Guardian"
+ grove_dryad_group:0 "Grove"
+ lord_male_dryad_group:0 "Lord"
+ lady_female_dryad_group:0 "Lady"
+ grand_grove_dryad_group:0 "Grand Grove"
+ high_lord_male_dryad_group:0 "High Lord"
+ high_lady_female_dryad_group:0 "High Lady"
+ domain_dryad_group:0 "Domain"
+ king_male_dryad_group:0 "King"
+ queen_female_dryad_group:0 "Queen"
+ kingdom_dryad_group:0 "Kingdom"
+
+ ### Tol'vir ###
+ # Feudal - Normal
+ governor_male_tolvir:0 "Governor"
+ governor_female_tolvir:0 "Governor"
+ region_tolvir:0 "Region"
+ nomarch_male_tolvir:0 "Nomarch"
+ nomarch_female_tolvir:0 "Nomarch"
+ nomes_tolvir:0 "Nomes"
+ pharaoh_male_tolvir:0 "Pharaoh"
+ pharaoh_female_tolvir:0 "Pharaoh"
+
+ # Feudal - Evil religions
+ dark_pharaoh_male_tolvir:0 "Dark Pharaoh"
+ dark_pharaoh_female_tolvir:0 "Dark Pharaoh"
+
+ ### Pandaren ###
+ # Feudal
+ leader_male_pandaren:0 "Leader"
+ leader_female_pandaren:0 "Leader"
+ district_pandaren:0 "District"
+ prefect_male_pandaren:0 "Prefect"
+ prefect_female_pandaren:0 "Prefect"
+ prefecture_pandaren:0 "Prefecture"
+ governor_male_pandaren:0 "Governor"
+ governor_female_pandaren:0 "Governor"
+ province_pandaren:0 "Province"
+
+ ### Mogu ###
+ # Feudal
+ leader_male_mogu:0 "Leader"
+ leader_female_mogu:0 "Leader"
+ district_mogu:0 "District"
+ lord_male_mogu:0 "Lord"
+ lady_female_mogu:0 "Lady"
+ commandery_mogu:0 "Commandery"
+ governor_male_mogu:0 "Governor"
+ governor_female_mogu:0 "Governor"
+ province_mogu:0 "Province"
+
+ ### Goblin ###
+ # Feudal / Republic
+ trade_lord_male_goblin:0 "Trade Lord"
+ trade_lady_female_goblin:0 "Trade Lady"
+ lordship_goblin:0 "Lordship"
+ trade_baron_male_goblin:0 "Trade Baron"
+ trade_baroness_female_goblin:0 "Trade Baroness"
+ high_barony_goblin:0 "High Barony"
+ trade_prince_male_goblin:0 "Trade Prince"
+ trade_princess_female_goblin:0 "Trade Princess"
+ trade_king_male_goblin:0 "Trade King"
```

Comment:
High Chieftain is the highest title that is canon for this title (https://wowpedia.fandom.com/wiki/Bilgewater_Cartel).